src/main/java/com/webonise/collectionAssignment/AList.java

```diff
@@ -0,0 +1,45 @@
+package com.webonise.collectionAssignment;
+
+import com.mongodb.BasicDBObject;
+import com.mongodb.DBCursor;
+
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Created by webonise on 19/9/16.
+ */
+public class AList implements ICollectionMenu {
```

Comment:
Please use proper class names, e.g AList could be ArrayList or something different for both understading and readability.